### DISCUSSION TITLE
Why does the program crash when running the program?

### DISCUSSION THREAD

**Discussion by jiadongzhen** [rifqiharrys/sdb_gui#6] [General]
I download the latest softwear of "sdb_gui_3.3.1_one_file.exe", the version is v3.3.1. And I uesd image and sample data provided in folder “data”.  The softwear crashed when click "Make Prediction" buttion. Why？

thank you very much

**Comment by rifqiharrys** [comment]
I'm sorry that you encounter that problem. I seem to have no problem using it. Did you select the correct depth header of the sample data?
![image](https://user-images.githubusercontent.com/24311199/182105140-8ae25387-02fa-4c21-8d21-d72e78fd5e94.png)
